<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>480</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Ferro Velho</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>3</number>
      </property>
      <widget class="QWidget" name="historico_page">
       <layout class="QGridLayout" name="gridLayout_2">
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QPushButton" name="DOUGLAS">
          <property name="styleSheet">
           <string notr="true">QPushButton::hover
{
background-color :rgb(186, 189, 182)
}</string>
          </property>
          <property name="text">
           <string>DOUGLAS</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="2">
         <widget class="QPushButton" name="GERAL">
          <property name="styleSheet">
           <string notr="true">QPushButton::hover
{
background-color :rgb(186, 189, 182)
}</string>
          </property>
          <property name="text">
           <string>GERAL</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QPushButton" name="ITO">
          <property name="styleSheet">
           <string notr="true">QPushButton::hover
{
background-color :rgb(186, 189, 182)
}</string>
          </property>
          <property name="text">
           <string>ITO</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QTableWidget" name="tableWidget_historico">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustIgnored</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>141</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>DATA</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PESO</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>MATERIAL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PREÃ‡O</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>HORA</string>
           </property>
          </column>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>TOTAL COMPRADO:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="label_historico_total">
          <property name="text">
           <string/>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="contabilidade_page">
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item row="0" column="0" colspan="3">
         <widget class="QFrame" name="frame_4">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>PESO:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="4">
            <widget class="QPushButton" name="pushButton_contabilidade_ok">
             <property name="styleSheet">
              <string notr="true">QPushButton::hover
{
background-color :rgb(114, 159, 207)
}</string>
             </property>
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
           <item row="1" column="3">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>VALOR POR KG</string>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QLineEdit" name="lineEdit_contabilidade_material"/>
           </item>
           <item row="2" column="3">
            <widget class="QLineEdit" name="lineEdit_contabilidade_valor"/>
           </item>
           <item row="1" column="2">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>MATERIAL:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEdit_contabilidade_peso"/>
           </item>
           <item row="2" column="5">
            <widget class="QPushButton" name="pushButton_contabilidade_limpar">
             <property name="styleSheet">
              <string notr="true">QPushButton::hover
{
background-color : rgb(186, 189, 182)
}</string>
             </property>
             <property name="text">
              <string>LIMPAR</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1" colspan="5">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>ABATER VENDA</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0" colspan="3">
         <widget class="QFrame" name="frame_5">
          <property name="frameShape">
           <enum>QFrame::NoFrame</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="2" column="0">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>TOTAL INVESTIDO:</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLabel" name="label_contabilidade_investido">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLabel" name="label_contabilidade_retornado">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>VALOR PRESO EM MATERIAL:</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>TOTAL RETORNADO:</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <widget class="QTableWidget" name="tableWidget_contabilidade">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAlwaysOff</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::NoSelection</enum>
             </property>
             <property name="columnCount">
              <number>5</number>
             </property>
             <attribute name="horizontalHeaderCascadingSectionResizes">
              <bool>true</bool>
             </attribute>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>141</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>false</bool>
             </attribute>
             <attribute name="verticalHeaderVisible">
              <bool>false</bool>
             </attribute>
             <column>
              <property name="text">
               <string>PESO</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>MATERIAL</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>VALOR</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>INVESTIDO</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>RETORNADO</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="4" column="1">
            <widget class="QLabel" name="label_contabilidade_lucro">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="precos_page">
       <layout class="QGridLayout" name="gridLayout_4">
        <property name="rightMargin">
         <number>15</number>
        </property>
        <item row="3" column="1">
         <widget class="QLineEdit" name="lineEdit_precos_novo_preco"/>
        </item>
        <item row="3" column="2">
         <widget class="QPushButton" name="pushButton_precos_ok">
          <property name="styleSheet">
           <string notr="true">QPushButton::hover
{
background-color :rgb(114, 159, 207)
}</string>
          </property>
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_11">
          <property name="text">
           <string>MUDAR PREÃ‡O</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLineEdit" name="lineEdit_precos_material"/>
        </item>
        <item row="3" column="3">
         <widget class="QPushButton" name="pushButton_precos_limpar">
          <property name="styleSheet">
           <string notr="true">QPushButton::hover
{
background-color : rgb(186, 189, 182)
}</string>
          </property>
          <property name="text">
           <string>LIMPAR</string>
          </property>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_12">
          <property name="text">
           <string>MATERIAL:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLabel" name="label_13">
          <property name="text">
           <string>NOVO PREÃ‡O:</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0" colspan="4">
         <widget class="QTableWidget" name="tableWidget_precos">
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="editTriggers">
           <set>QAbstractItemView::NoEditTriggers</set>
          </property>
          <property name="selectionMode">
           <enum>QAbstractItemView::NoSelection</enum>
          </property>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>350</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>MATERIAL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PREÃ‡O</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="relatorios_page"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QFrame" name="frame">
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_historico">
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
background-color :rgb(186, 189, 182)
}</string>
         </property>
         <property name="text">
          <string>HISTÃ“RICO</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_contabilidade">
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
background-color :rgb(186, 189, 182)
}</string>
         </property>
         <property name="text">
          <string>CONTABILIDADE</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_precos">
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
background-color :rgb(186, 189, 182)
}</string>
         </property>
         <property name="text">
          <string>PREÃ‡OS</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="text">
          <string>RELATÃ“RIOS</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_login_ito">
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
background-color :rgb(50, 133, 128)
}</string>
         </property>
         <property name="text">
          <string>ITO</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_login_douglas">
         <property name="styleSheet">
          <string notr="true">QPushButton::hover
{
background-color :rgb(50, 133, 128)
}
</string>
         </property>
         <property name="text">
          <string>DOUGLAS</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
